vault/dendron.topic.publish.migration.md

```diff
@@ -2,10 +2,15 @@
 id: rYbs1qLh9VJBXCJlSzMt4
 title: Migration
 desc: ''
-updated: 1645151071750
+updated: 1645155025898
 created: 1632351743935
 ---
 
+## Upgrade dendron CLI
+Please note that if you have an automated pipeline set up for publishing that uses `dendron-cli`, you need to manually upgrade `dendron-cli` that is compatible to the current configuration version to avoid errors due to configuration mismatch.
```

Comment:
```suggestion
Please note that if you have an automated pipeline set up for publishing that uses `dendron-cli`, you need to manually upgrade `dendron-cli` so that is compatible with the current configuration version. Upgrading helps avoid errors due to configuration mismatch.
```